[{"count": 0, "attachment_id": null, "creator": "knitterb@blandsite.org", "text": "I'm running 20 threads each on my two remote JMeter Server instances from the JMeter GUI.  I get the following exception on the JMeter Server instances after about 2-3 minutes.  Is this normal?\n\n\n\n2008/04/03 16:59:04 ERROR - jmeter.samplers.BatchSampleSender: sampleOccurred java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: \n\tjava.io.EOFException\n\tat sun.rmi.transport.tcp.TCPChannel.createConnection(Unknown Source)\n\tat sun.rmi.transport.tcp.TCPChannel.newConnection(Unknown Source)\n\tat sun.rmi.server.UnicastRef.invoke(Unknown Source)\n\tat org.apache.jmeter.samplers.RemoteSampleListenerImpl_Stub.processBatch(Unknown Source)\n\tat org.apache.jmeter.samplers.BatchSampleSender.SampleOccurred(BatchSampleSender.java:131)\n\tat org.apache.jmeter.samplers.RemoteListenerWrapper.sampleOccurred(RemoteListenerWrapper.java:86)\n\tat org.apache.jmeter.threads.ListenerNotifier.notifyListeners(ListenerNotifier.java:240)\n\tat org.apache.jmeter.threads.JMeterThread.notifyListeners(JMeterThread.java:576)\n\tat org.apache.jmeter.threads.JMeterThread.run(JMeterThread.java:313)\n\tat java.lang.Thread.run(Unknown Source)", "id": 115309, "time": "2008-04-03T17:00:00Z", "bug_id": 44753, "creation_time": "2008-04-03T17:00:00Z", "tags": [], "is_private": false}, {"count": 1, "tags": [], "text": "Not seen the problem, but it looks like the Java library (rather than JMeter) has detected a problem, possibly network-related.\n\nHave you got a simple test case that can be used to reliably demonstrate the problem?", "is_private": false, "bug_id": 44753, "id": 116412, "time": "2008-05-09T02:03:32Z", "creator": "sebb@apache.org", "creation_time": "2008-05-09T02:03:32Z", "attachment_id": null}, {"count": 2, "tags": [], "text": "(In reply to comment #1)\n> Not seen the problem, but it looks like the Java library (rather than JMeter)\n> has detected a problem, possibly network-related.\n> \n> Have you got a simple test case that can be used to reliably demonstrate the\n> problem?\n> \n\nI don't really have a test case. I was actually just testing against a MediaWiki link (just one URL, not a scenario) and had it loaded up to about 200 concurrent connections over two machines (100 connections each).\n\nIt seems that when JMeter is under great load link this that it's fairly reproducable.  Maybe a limitation of the rmiregistry process under load?", "is_private": false, "bug_id": 44753, "id": 116428, "time": "2008-05-09T11:48:53Z", "creator": "knitterb@blandsite.org", "creation_time": "2008-05-09T11:48:53Z", "attachment_id": null}, {"count": 3, "tags": [], "text": "It does not look like there is anything that JMeter can do to prevent this occurring.", "is_private": false, "bug_id": 44753, "id": 116581, "time": "2008-05-13T14:53:09Z", "creator": "sebb@apache.org", "creation_time": "2008-05-13T14:53:09Z", "attachment_id": null}]